docs/root/api-docs/xds_protocol.rst

```diff
@@ -384,13 +384,19 @@ as well as a mechanism to ACK/NACK configuration updates.
 ACK
 ^^^
 
-If the update was successfully applied, the
+If the update contained valid resources, the
 :ref:`version_info <envoy_v3_api_field_service.discovery.v3.DiscoveryRequest.version_info>` will be ``X``, as indicated
 in the sequence diagram:
 
 .. figure:: diagrams/simple-ack.svg
    :alt: Version update after ACK
 
+.. note::
+
+   If an xDS client responds with an ACK, it does _not_ mean that a configuration was successfully
+   applied. ACKing an xDS resource simply means that when evaluated in isolation, the xDS resource
```

Comment:
Just to be completely clear here, although the ACK is based on the validity of the received resources, the thing we're actually ACKing in the response, not the individual resource.  So I suggest changing this sentence to something like this:

ACKing simply means that the client considered each of the resources in the response to be valid when evaluated in isolation.